.travis.yml

```diff
@@ -1,25 +1,29 @@
-sudo: required
-dist: trusty
 language: node_js
 
+sudo: required
+
+os:
+  - osx
+
+# See https://docs.travis-ci.com/user/reference/osx/#macos-version
+# This implies macOS 10.14
+osx_image: xcode11
+
 cache:
   directories:
-    - "$HOME/.npm"
```

Comment:
What's the story here?

Reply:
As per https://docs.travis-ci.com/user/caching/#npm-cache, it should happen automatically now.